### PR TITLE
🌴 Khe Sanh port

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/Map/vn_khe_sanhInfo.sqf [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+		//[_popValue, _disableTownName];
+		[_popValue, _disableTownName];
+	};
+	case "antennas": {
+		private _posAntennas =
+		[[9048.66,6549.03,-0.258987], [9387.63,10437.2,0.00366211], [5117.96,11335,-110.99], [2792.38,8592.84,0.140808],
+		[10692.6,3183.53,-0.000190735], [11182.8,1808.63,0.138306], [4562.41,866.129,0.49115], [748.892,11575.5,0.138336],
+		[11235.1,14857.8,0.000335693], [412.002,509.969,0.00694847]];
+		private _blacklistIndex = [];
```
Comment (killerswin2) [inline Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/Map/vn_khe_sanhInfo.sqf:53]: no blacklist, standard is to nil, but okay

**Review by killerswin2 — APPROVED** [review]

File: Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/mission.sqm [hunk 2]
```diff
@@ -0,0 +1,12882 @@
...
+						};
+						class Item17
+						{
+							dataType="Marker";
+							position[]={3806.479,2.7037606e+012,5781.145};
+							name="respawn_guerrila";
+							text="Your Headquarters";
+							type="hd_flag";
+							colorName="ColorGUER";
+							angle=62.260929;
```
Comment (Lazejun) [inline Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/mission.sqm]:
```suggestion
							angle=0;
```

Comment (Lazejun) [inline Map-Templates/Antistasi-Khe-Sanh.vn_khe_sanh/mission.sqm]:
```suggestion
							angle=0;
```

**Review by Lazejun — CHANGES_REQUESTED** [review]
grafik

"airport_2" we either remove the wall infront of the Planes or tell SGD to turn around the Hangars

**Comment by Bob-Murphy** [comment]
Implemented all changes into `map-overhaul` branch.
Hence this PR will be closed.

**Pull request closed without merging**